ouroboros-network/test/Test/LedgerPeers.hs

```diff
@@ -58,6 +61,27 @@ instance Arbitrary ArbitraryRelayAccessPoint where
               , RelayAccessDomain  "relay.iohk.example" . getArbitraryPortNumber <$> arbitrary
               ]
 
+newtype ArbitraryLedgerStateJudgement =
+  ArbitraryLedgerStateJudgement {
+    getArbitraryLedgerStateJudgement :: LedgerStateJudgement
+  } deriving Show
+
+instance Arbitrary ArbitraryLedgerStateJudgement where
+    arbitrary =
+      ArbitraryLedgerStateJudgement <$>
+        oneof [ pure YoungEnough
+              , pure TooOld
+              ]
+
+newtype ArbitrarySlotNo =
+  ArbitrarySlotNo {
+    getArbitrarySlotNo :: SlotNo
+  } deriving Show
+
+instance Arbitrary ArbitrarySlotNo where
+    arbitrary =
+      ArbitrarySlotNo . fromInteger <$> arbitrary
```

Comment:
This relies on `Arbitrary Int` instance, which can generate negative numbers.  Is this fine?

Reply:
For the purposes of this test it is.

Reply:
ok. Please add a comment, e.g. something like:
> We generate integers including negative ones, which is fine for the purpose of this test.